packages/backend/src/handlers/remote-schemas/typeDefs.ts

```diff
@@ -1,9 +1,19 @@
 import { gql } from 'graphql-request/dist';
 
 export const typeDefs = gql`
+  scalar uuid
```

Comment:
Without this Hasura was giving me an error when creating a remote relationship.
See hasura/graphql-engine#5482